### ISSUE TITLE
Use Implicit Type refactoring should detect type as apparent in declaration for ValueTuple

### ISSUE THREAD

**Issue by jcouv** [dotnet/roslyn#11154]
See [background](https://github.com/dotnet/roslyn/pull/10994#discussion_r62239147).

This test broke after the tuple symbol has been updated to unify tuple and ValueTuple. 

``` C#
        [WpfFact, Trait(Traits.Feature, Traits.Features.CodeActionsUseImplicitType)]
        public async Task ValueTupleCreate()
        {
            await TestAsync(
@"using System; class C { static void M() { [|ValueTuple<int, int>|] s = ValueTuple.Create(1, 1); } }" + trivial2uple,
@"using System; class C { static void M() { var s = ValueTuple.Create(1, 1); } }" + trivial2uple,
options: ImplicitTypeWhereApparent(),
parseOptions: TestOptions.Regular.WithTuplesFeature(),
withScriptOption: true);
        }
```

File is `UseImplicitTypeTests.cs`.

FYI for @balajikris

**Comment by DustinCampbell** [comment]
Moving to compiler since it's assigned to @jcouv. Let us know if IDE should take a look.

**Comment by jcouv** [comment]
@DustinCampbell What's your thoughts on priority? 
Given that the common scenario will be to use tuple syntax and not `ValueTuple`, I'm moving this out of 2.1.

**Comment by DustinCampbell** [comment]
That makes sense to me. Adding @Pilchie as FYI.
